config/env.config.js

```diff
@@ -59,8 +59,10 @@ const RedirectPlugin = function(options) {
     this.apply = function(compiler) {
         compiler.plugin('compilation', function(compilation) {
             compilation.plugin('html-webpack-plugin-before-html-processing', function(htmlPluginData, callback) {
-                let htmlHead = htmlPluginData.html.match('<head>').index;
-                htmlPluginData.html = htmlPluginData.html.slice(0, htmlHead) + '<head>' +
+                let htmlHead = htmlPluginData.html.match('<head>');
+                if (htmlHead) {
+                    htmlHead = htmlHead.index;
+                    htmlPluginData.html = htmlPluginData.html.slice(0, htmlHead) + '<head>' +
                     `<script>
                         var getParameterByName = function(name) {
```

Comment:
Should this whole script be wrapped in a self-executing function, to avoid polluting the namespace?

     (function() {
            ...
     })();